README.adoc

```diff
@@ -107,154 +92,157 @@ mvn liberty:stop-server
-is being used, a status of `DOWN` will be returned.  
+is being used, a status of `DOWN` will be returned.
 
 // =================================================================================================
 // Adding health checks to the inventory service
 // =================================================================================================
 
 === Adding health checks to the inventory service
 
-[role="code_command hotspot file=0", subs="quotes"] 
----- 
+[role="code_command hotspot file=0", subs="quotes"]
+----
 #Create the `InventoryReadinessCheck` class.#
 `src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java`
----- 
+----
 
 InventoryReadinessCheck.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright tags=InventoryReadinessCheck']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[tags=InventoryReadinessCheck]
+include::finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[]
 ----
 
-This time, you are checking whether or not the service is in maintenance or if it's down. 
-For simplicity, the custom [hotspot=3 file=2]`io_openliberty_guides_inventory_inMaintenance` 
-MicroProfile Config property defined in the [hotspot file=2]`resources/CustomConfigSource.json` 
-file is used to indicate whether the service is in maintenance or not. This file has already 
-been created for you. To check if the service is down, simply make a HTTP GET request to 
-the `system` service and check the status returned by the response. You make a GET request 
-to the `system` service rather than the `inventory` service because the `inventory` service 
-depends on the `system` service. In other words, the `inventory` service wouldn't work if 
-the `system` service is down. If the status is not 200, then the service is not running. 
-Based on these two factors, the [hotspot=19-36 file=0]`isHealthy()` method returns whether 
+This time, you are checking whether or not the service is in maintenance or if it's down.
+For simplicity, the custom [hotspot=3 file=2]`io_openliberty_guides_inventory_inMaintenance`
+MicroProfile Config property defined in the [hotspot file=2]`resources/CustomConfigSource.json`
+file is used to indicate whether the service is in maintenance or not. This file has already
+been created for you. To check if the service is down, simply make a HTTP GET request to
+the `system` service and check the status returned by the response. You make a GET request
+to the `system` service rather than the `inventory` service because the `inventory` service
+depends on the `system` service. In other words, the `inventory` service wouldn't work if
+the `system` service is down. If the status is not 200, then the service is not running.
+Based on these two factors, the [hotspot=isHealthy file=0]`isHealthy()` method returns whether
```

Comment:
Update this paragraph to:
"This time, you are checking whether the service is in maintenance or if it's down. For simplicity, the custom `io_openliberty_guides_inventory_inMaintenance` MicroProfile Config property that is defined in the `resources/CustomConfigSource.json` file is used to indicate whether the service is in maintenance or not. This file was already created for you. To check whether the service is down, make an HTTP GET request to the `system` service and check the status that is returned by the response. You make a GET request to the `system` service rather than the `inventory` service because the `inventory` service depends on the `system` service. In other words, the `inventory` service doesn't work if the `system` service is down. If the status is not `200`, then the service is not running. Based on these two factors, the `isHealthy()` method returns whether the `inventory` service is healthy."

Reply:
Updated